### PR TITLE
chore(environments): Update scripts for environments split

### PR DESCRIPTION
**Pull request by wlee221**
#### Description of changes

Updates `package.json` `scripts` and `pull-environments.sh` to reflect directory changes in #2501.

#### Description of how you validated changes

#2505 is running test against all environment split changes.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: environments/pull-environments.sh [hunk 1]
```diff
@@ -21,5 +21,5 @@ if [ "$NODE_ENV" = "test" ]; then
   numParallelTasks=1; # GitHub actions has trouble handling parallel executions
 fi
 
-echo $dirs | xargs -P $numParallelTasks -I {} sh -c "./pull-environment.sh {}"
+echo $dirs | xargs -P $numParallelTasks -I {} sh -c "../pull-environment.sh {}"
```
Comment (slaymance) [inline environments/pull-environments.sh:24]: I understand this is relative to the cwd of each environment folder, but it is a little confusing that we're referencing a file in the same directory as this script using `..`. I wonder if it's easy to reference `pull-environment.sh` from the path of `pull-environments.sh`.

Comment (wlee221) [reply environments/pull-environments.sh:24]: That's a good point; we can get the directory that `pull-environments.sh` is in with:

```bash
shell_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
```

and then we can do:

```suggestion
echo $dirs | xargs -P $numParallelTasks -I {} sh -c ""$shell_path"/pull-environment.sh {}"
```

Comment (wlee221) [reply environments/pull-environments.sh:24]: (source: https://stackoverflow.com/a/24112741)

Comment (wlee221) [reply environments/pull-environments.sh:24]: actually that shell command can prob be simplified... taking a look.

Comment (slaymance) [reply environments/pull-environments.sh:24]: I this that approach is a bit cleaner and allows us to run the `pull-environments.sh` script from any directory, but I'd say timebox it as I think it's just a nice-to-have :)

Comment (wlee221) [reply environments/pull-environments.sh]: Agreed, done! [d506c78](https://github.com/aws-amplify/amplify-ui/pull/2502/commits/d506c7871cc97c404366ff67ccdab135918d9224)

**Review by slaymance — APPROVED** [review]
🚀

**Review by reesscot — APPROVED** [review]
LGTM

**Pull request merged**